app/tests/test_gemini_service.py

```
@@ -0,0 +1,499 @@
+                result = None
+                for _ in range(5):
+                    try:
+                        result = service.ask("retry test")
+                        if result:
+                            break
+                    except Exception:
+                        continue
+
+                total_time = time.time() - start_time
+
+                # リトライ間でもレートリミットが守られることを確認（時間は短縮）
```

Ruff (0.14.1)

Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)